src/Symfony/Component/HttpFoundation/UriSigner.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\HttpFoundation;
 
+use Symfony\Component\Clock\ClockInterface;
```

Comment:
Can we use the PSR ClockInterface instead ? Usages in this class should be compatible with it, and it makes the class easier to use outside the fullstack framework (other frameworks might use a different clock implementation).
That's similar to how our components dispatching events are typing against the PSR-14 interface instead of `symfony/event-dispatcher` in most cases.